Makefile

```diff
@@ -2,3 +2,6 @@ all:
 	mkdir -p ./bin && \
 		go build -o bin/openshift-must-gather cmd/must-gather.go && \
 		go build -o bin/openshift-dev-helpers dev-helpers/cmd/dev-helpers.go
+
+update-deps:
```

Comment:
This will eventually have to go as soon as we switch to @tnozicka's build system

Reply:
yep, agree